jekyll/features.html

```diff
@@ -305,25 +305,15 @@ <h2 id="easy-c-obj-c-wrapping">Easy C, C++ <b>and</b> Objective C wrapping</h2>
   Event* {.importcpp: "sf::Event".} = object
 
 {.push cdecl, header: "<SFML/Graphics.hpp>".}
+{.push importcpp.}
```

Comment:
Did you test this?

Reply:
No, but I don't think this has to be a full example that actually compiles.

Reply:
Now I did. It compiles.